packages/bits/src/lib/textbox/textbox-number/textbox-number.component.less

```
@@ -7,6 +7,7 @@
 @textbox-padding-simple-updown: @textbox-updown-offset + @textbox-feedback-icon-margin;
 @textbox-feedback-icon-with-updown-margin: @textbox-feedback-icon-margin + @textbox-updown-offset;
 @textbox-busy-offset: 1px;
+@textbox-button-height: 28px;
```

This can be improved by calculating the correct height of the buttons by subtracting the border (times 2) from input height base value.

Like this:

```suggestion
@textbox-button-height: @input-height-base - (@nui-line-default * 2);
```

Here's where the form control styling is set giving it the height via a direct rule and a border via the textbox-base() mixin:

https://github.com/solarwinds/nova/blob/891476fc40a072e0725ced68a786da2d9d143577/packages/bits/src/styles/components/forms.less#L35

---

Fixed. Thank you @Blakesalot for the detailed description